common/src/main/java/me/shedaniel/architectury/event/events/BlockEvent.java

```diff
@@ -32,32 +32,56 @@
 import org.jetbrains.annotations.Nullable;
 
 public interface BlockEvent {
-    
-    // Block interaction events
     /**
-     * Called when a player breaks a block.
+     * @see Break#breakBlock(Level, BlockPos, BlockState, ServerPlayer, IntValue)
      */
     Event<Break> BREAK = EventFactory.createInteractionResult();
     /**
-     * Called when a block is placed in the world by an entity.
+     * @see Place#placeBlock(Level, BlockPos, BlockState, Entity)
      */
     Event<Place> PLACE = EventFactory.createInteractionResult();
-    
     /**
-     * Called when a falling block (sand, anvil, etc.) is about to land.
-     * Use fallState#getBlock to get the type of block for more granular control.
+     * @see FallingLand#onLand(Level, BlockPos, BlockState, BlockState, FallingBlockEntity)
      */
     Event<FallingLand> FALLING_LAND = EventFactory.createLoop();
     
     interface Break {
+        /**
+         * Invoked when a block is destroyed by a player.
+         *
+         * @param world  The level the block is in.
+         * @param pos    The position of the block.
+         * @param state  The current state of the block.
+         * @param player The player who is breaking the block.
+         * @param xp     The experience that are dropped when the block was destroyed. Always null on fabric!
+         * @return Return {@link InteractionResult#FAIL} to cancel the block breaking.
+         */
         InteractionResult breakBlock(Level world, BlockPos pos, BlockState state, ServerPlayer player, @Nullable IntValue xp);
     }
     
     interface Place {
+        /**
+         * Invoked when a block is placed.
+         *
+         * @param world  The level the block is in.
+         * @param pos    The position of the block.
+         * @param state  The future state of the block.
+         * @param placer The entity who is placing it. Can be null, when the placer isn't a entity.
```

Comment:
Please apply to all nulls
```suggestion
         * @param placer The entity who is placing it. Can be {@code null}, when the placer isn't a entity.
```